cardano-node/src/Cardano/CLI/Tx/Generation.hs

```diff
@@ -115,6 +116,10 @@ newtype TPSRate =
   TPSRate Int
   deriving (Eq, Ord, Show)
 
+newtype TxAdditionalSize =
```

Comment:
Can you add a comment elaborating what `TxAdditionalSize` is?

Reply:
Yes, I'll add.